test/lib/classify/utilities_test.rb

```diff
@@ -13,6 +13,18 @@ def test_supported_value
     refute Primer::Classify::Utilities.supported_value?(:m, :foo)
   end
 
+  def test_supported_selector
+    assert Primer::Classify::Utilities.supported_selector?("m-1")
+    refute Primer::Classify::Utilities.supported_selector?("foo")
+  end
+
+  def test_supported_selector_returns_nil_in_production
+    Rails.stub(:env, ActiveSupport::StringInquirer.new("production")) do
+      assert_nil Primer::Classify::Utilities.supported_selector?("m-1")
```

Comment:
Generally, Ruby predicate methods should always return a boolean. Perhaps we could update this to `return false` in the early return case?